tox.ini

```diff
@@ -1,7 +1,6 @@
 [tox]
 envlist = py34-pytest2,py34-pytest3,py34,py35,py36,py37,pep8
```

Comment:
Could you please remove also `py34-pytest2,py34-pytest3,py34` from here ?

In addition, we can get rid of lines which are describing these targets down bellow:
```
[testenv:py34-pytest2]
...
[testenv:py34-pytest3]
...
```

And then we can remove also files  `requirements-pytest2.txt` & `requirements-pytest3.txt` !

Reply:
done